packages/core/src/formats.ts

```diff
@@ -24,6 +25,23 @@ export function date(
   return formatter.format(isString(value) ? new Date(value) : value)
 }
 
+export function relative(
+  locales: Locales,
+  value: number,
+  unit: Intl.RelativeTimeFormatUnit,
+  format?: Intl.RelativeTimeFormatOptions
+): string {
+  if ((isDate(value) || isString(value)) && unit) throw new Error('with units you should use numbers as values')
```

Comment:
I'm not a maintainer, so take this only as a recommendation, but I think the error message can be improved a little. eg. it could contain the `value`.

Reply:
I removed error message, I planned to make `relative` accept date or string and generate (value, unit) from it, I will ignore that for now.